ush/python/pygfs/task/atmens_analysis.py

```diff
@@ -105,6 +107,80 @@ def initialize(self: Analysis) -> None:
         ]
         FileHandler({'mkdir': newdirs}).sync()
 
+    @logit(logger)
+    def observer(self: Analysis) -> None:
+        """Execute a global atmens analysis observer
+
+        This method will execute a global atmens analysis observer using JEDI.
+        This includes:
+        - changing to the run directory
+        - running the global atmens analysis executable in observer mode
+
+        Parameters
+        ----------
+        Analysis: parent class for GDAS task
+
+        Returns
+        ----------
+        None
+        """
+        chdir(self.task_config.DATA)
+
+        exec_cmd = Executable(self.task_config.APRUN_ATMENSANLOBS)
+        exec_name = os.path.join(self.task_config.DATA, 'gdas.x')
+
+        exec_cmd.add_default_arg(exec_name)
+        exec_cmd.add_default_arg('fv3jedi')
+        exec_cmd.add_default_arg('localensembleda')
+        exec_cmd.add_default_arg(self.task_config.jedi_yaml)
+
+        try:
+            logger.debug(f"Executing {exec_cmd}")
+            exec_cmd()
+        except OSError:
+            raise OSError(f"Failed to execute {exec_cmd}")
+        except Exception:
+            raise WorkflowException(f"An error occured during execution of {exec_cmd}")
+
+        pass
+
+    @logit(logger)
+    def solver(self: Analysis) -> None:
```

Comment:
I think this should be `solve` instead of `solver`. Methods should generally be verbs.

Reply:
DA developers view _observer_ and _solver_ as functional steps (that is, nouns) in the DA process.  The names were chosen to make things more readable to DA developers.  I wasn't thinking about naming conventions for python methods.

I'm fine with replacing _solver_ and _solve_ and _observer_ with _observe_ if this is what the g-w team wants.

Reply:
Replace _solver_ with _solve_ and _observer_ with _observe_ in the following
- `scripts/exglobal_atmens_analysis_obs.py`
- `scripts/exglobal_atmens_analysis_sol.py`
- `ush/python/pygfs/task/atmens_analysis.py`

Done at 28e6e5a.